frame/nomination-pools/src/tests.rs

```diff
@@ -30,6 +30,13 @@ macro_rules! unbonding_pools_with_era {
 	}};
 }
 
+macro_rules! typed_bounded_btree_map {
```

Comment:
This name is ambiguous because it doesn't say the type. IMO something like `delegator_unbonding_eras` makes more sense

Reply:
d0f0121c68d30df69f463c7c51b1a38c09e34080